pipelines/broad/arrays/imputation_beagle/test_inputs/Plumbing/test_data_NA12878_x50.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ImputationBeagle.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation_beagle/scientific/vcfs/NA12878_50_duplicate.clean.vcf.gz",
+    "ImputationBeagle.ref_dict": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
+    "ImputationBeagle.reference_panel_path_prefix": "gs://broad-gotc-test-storage/imputation_beagle/scientific/1000G_HGDP_no_singletons_reference_panel/hgdp.tgp.gwaspy.AN_added.bcf.ac2",
+    "ImputationBeagle.contigs": ["chr21","chr22"],
```

Comment:
ah cool so plumbing is less contigs, that makes sense

Reply:
yeah i think that'll be ok. if it takes too long we could consider a subselection of the chromosome, which we'd have to do on the ref panel files too